DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/LookUpTable.h

```diff
@@ -228,10 +228,9 @@ class LookUpTable
     std::cout << "lut_data.size " << lut_data.size() << std::endl;
     return o2::ft0::LookUpTable{lut_data};
   }
-  bool isTCM(int link, int ep)
-  {
-    return getChannel(link, 1, ep) == TCM_channel;
-  }
+  bool isTCM(int link, int ep) const { return getChannel(link, 1, ep) == TCM_channel; }
+  Topo getTopoPM(int globalChannelID) const { return mTopoVector[globalChannelID]; }
+  Topo getTopoTCM() const { return mTopoVector[TCM_channel]; }
```

Comment:
do you really want to return a copy or the reference would be enough?

Reply:
@shahor02 I removed this lines. Code is fixed, CI seems to be okay.